.github/workflows/homebrew.yml

```
@@ -32,41 +19,182 @@ jobs:
+              cat asd.phar
+              exit 1
+            fi
+          else
+            echo "❌ Failed to download asd.phar"
+            exit 1
+          fi
 
       - name: Calculate SHA256
+        if: env.PHAR_EXISTS == 'true'
         id: sha
         run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)